default/scripting/policies/ISOLATION.focs.txt

```diff
@@ -21,17 +21,20 @@ Policy
                 VisibleToEmpire empire = Source.Owner
             ]
             effects =
-                SetTargetHappiness value = Value + min(
-                    10.0, 
-                    (Statistic Min
-                        value = DirectDistanceBetween object = Target.SystemID object = LocalCandidate.SystemID
-                        condition = And [
-                            Planet
-                            Species
-                            VisibleToEmpire empire = Source.Owner
-                            Not Target
-                        ]
-                    ) / (NamedReal name = "PLC_ISOLATION_THRESHOLD_DISTANCE" value = 150.0) * 10.0 - 10.0)
+                SetTargetHappiness value = Value + max(
```

Comment:
a bit more refactoring going on here that the description suggests...

Reply:
Original is -10 at 0uu(or in-system), +1/15uu step. +0 at 150uu, +10 at 300uu
Modified version is -5 at within 50uu. then +1/10uu step. +0 at 100uu, +5 at 150uu

At -10, 2 planets in the same system is virtually impossible (they can be on defense focus). That's more enforced isolation, and I doubt anyone will take it under those restrictions.

At -5, 2 planets in the same system is still possible, just under a significant penalty. It's more an empire that is "mostly isolated" but can still have a shared system.

I assume you noticed that there was also a change where the stealth bonus works on outposts? There's been a long running complaint that the stealth races(Laenfa, Sly, Etty and Trith) have no real way to hide outposts.

Reply:
I'd also be fine with -5 within 60uu, +0 at 120uu and +5 at 180uu.
Or something else if someone has a better suggestion